### PR TITLE
Update examples for @userfront/toolkit v1.0.2

### PR DESCRIPTION
**Pull request by RJFelix**
Update these examples to use `@userfront/toolkit v1.0.2` rather than the legacy framework-specific libraries:

- NextJS
- NextJS w/FastAPI
- React
- Vue 2

### CODE REVIEW

File: react/src/App.js [hunk 1]
```diff
@@ -7,20 +7,10 @@ import {
   Navigate,
   useLocation,
 } from "react-router-dom";
-import Userfront from "@userfront/react";
+import Userfront, { SignupForm, LoginForm, PasswordResetForm } from "@userfront/toolkit";
```
Comment (drbrady8800) [inline react/src/App.js]: Does this have to be `@userfront/toolkit/react`?

File: vue2/README.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# userfront-vue-2-example
+
+This template should help get you started developing with Vue 3 in Vite.
```
Comment (drbrady8800) [inline vue2/README.md]: The title and the description mismatch - is it Vue 2 or Vue 3 or is it both?

**Review by drbrady8800 — COMMENTED** [review]
Looks good, just those two clarification comments.

File: react/src/App.js (hunk 1 above)
Comment (RJFelix) [reply react/src/App.js]: Good catch!

File: vue2/README.md (hunk 2 above)
Comment (RJFelix) [reply vue2/README.md]: It's Vue 2.

**Review by drbrady8800 — APPROVED** [review]
Looks good!

**Pull request merged**